### PR TITLE
LNX-126 Add displaying tree object

### PR DESCRIPTION
**Pull request by Krystian030**

### CODE REVIEW

**Review by alpinus4 — COMMENTED** [review]
LGTM

File: lynx/common/actions/chop.py [hunk 1]
```diff
@@ -1,30 +1,41 @@
...
-		for object in objects_on_square:
-			# TODO: put all object names into an Enum
-			if object.name == 'Tree':
-				remove_action = RemoveObject(object.id)
-				log = Object(id=scene.generate_id(), name='Log', position=object.position, pickable=True)
-				create_action = CreateObject(log.serialize())
-				scene.add_to_pending_actions(remove_action.serialize())
-				scene.add_to_pending_actions(create_action.serialize())
+    def satisfies_requirements(self, scene: 'Scene') -> bool:
+        return CommonRequirements.is_in_range(scene, self.object_id, self._get_target_position(scene), 1) \
```
Comment (EasyJB) [inline lynx/common/actions/chop.py:37]: Do we need this requirement, if we use Directions?

**Review by EasyJB — COMMENTED** [review]
Imo ok, just one question

File: lynx/common/actions/chop.py (hunk 1 above)
Comment (Krystian030) [reply lynx/common/actions/chop.py:37]: Generally, if we want, we can also pass any vector to chop(), so I think it's necessary.

**Pull request merged**